pulp_ansible/tests/functional/api/collection/v3/test_serializers.py

```diff
@@ -0,0 +1,94 @@
+# coding=utf-8
+"""Tests related to sync ansible plugin collection content type."""
```

Comment:
Maybe this docstring gets updated to focus on serializers?

Reply:
I forgot the docstrings, I'll fix it